client/client/main.go

```diff
@@ -140,12 +162,15 @@ func main() {
 	reader := bufio.NewReader(os.Stdin)
 
 	for {
-		fmt.Print("Please enter word to search for (enter to exit): ")
-		keyword, _ := reader.ReadString('\n')
-		keyword = keyword[:len(keyword)-1]
-		if keyword == "" {
+		fmt.Print("Please enter words to search for separated by spaces (enter to exit): ")
+		input, _ := reader.ReadString('\n')
+		input = strings.TrimRight(input, "\n")
+		if input == "" {
 			break
 		}
-		performSearchWord(cli, keyword)
+		keywords := strings.Split(input, " ")
+		for _, keyword := range keywords {
+			performSearchWord(cli, keyword)
```

Comment:
This is likely to be pretty confusing, because the results don't give any indication of what keyword they are for.  You're effectively doing an "OR" query, when I think most people would expect an "AND" query.  If we can't easily do an "AND" query, I'd rather just restrict it to querying one word at a time and print an error if there are multiple keywords.